code-challenges.test.js

```diff
@@ -36,7 +45,29 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+describe("maybeDivisibleByThree", () => {
+    it("determine if the number stated is divisible by three", () => {
+        expect(maybeDivisibleByThree("yes")).toEqual("is divisible by three")
+        expect(maybeDivisibleByThree("no")).toEqual("is not divisible by three")
+    })
+})
+
+const maybeDivisibleByThree = (string) => {
+    if (string === "yes") {
+        return "15 is divible by three", "0 is divisible by three"
+    } else if (string === "no") {
+        return "-7 is not divisible by three"
+    }
+}
+})
+
 // Pseudo code:
+// name of function: maybeDivisibleByThree
+// input: 3 objects as an argument
+// output: 3 strings that determine if the arguments listed are divible by 3 or not. 
+// process: following the steps provided in the syllabus to create a test with arguments
+
+// NOTES: This one has failed too. I thought that by setting up the arguments I could get the desired outputs - maybe I've missed a step within the "const section". I believe I correctly organized the "describe, it, expect" portion of the test. The terminal says "error Command failed with exit code 1"
```

Comment:
Your syntax looks great! The issue is in the expect statement. The expect statement performs a function call `maybeDivisibleByThree("yes")`. However, the prompt is stating that the function should take an object as an argument. In this expect statement, you are giving the function a string `yes` as an argument.